app/views/users/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<!-- <%= form_with model: @user, url: edit_user_path(@user), method: :patch do |form| %>
+  <br>
+  <%= form.label :username, "Edit Username:" %>
+  <%= form.text_field :username, :value => @user.username %><br>
+  <%= form.label :email, "Edit Email:" %>
+  <%= form.text_field :email, :value => @user.email %><br>
+  <%= form.label :password, "Edit Password:" %>
+  <%= form.password_field :password %><br>
+  <%= form.submit "Submit" %>
+<% end %>
+<br>
+<%= link_to "<< Back to Users List", root_path %><br/> -->
+<h1>Edit User</h1>
+
+<%= render 'form', user: @user %>
```

Comment:
This file should still contain all the commented-out forms used in the new.html file

([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_edit.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))